data/SinghRamesh_2019/metadata.yml

```diff
@@ -0,0 +1,353 @@
+    in storage bags, moistened with damp tissues. Leaves were weighed to obtain fresh
+    mass and oven dried for a about 42-78 h at 60℃ and its oven-dried mass were calculated.
+  original_file: in raw data folder
+  notes: none
+locations:
+  Mount Bellenden Ker_1:
+    elevation (m): 1539.0
+    latitude (deg): -17.26336
+    longitude (deg): 145.85396
+  Mount Bellenden Ker_2:
+    elevation (m): 1544.0
+    latitude (deg): -17.26497
+    longitude (deg): 145.8535
+  Mount Bellenden Ker_3:
+    elevation (m): 1552.0
+    latitude (deg): -17.26497
+    longitude (deg): 145.8535
+  Mt Bellenden Ker_1:
+    elevation (m): 1532.0
+    latitude (deg): -17.26334
+    longitude (deg): 145.85397
+  Mt Bellenden Ker_2:
+    elevation (m): 1539.0
+    latitude (deg): -17.26336
+    longitude (deg): 145.85396
+  Mt Bellenden Ker_3:
+    elevation (m): 1544.0
+    latitude (deg): -17.26497
+    longitude (deg): 145.8535
+  Mt Bellenden Ker_4:
+    elevation (m): 1550.0
+    latitude (deg): -17.26497
+    longitude (deg): 145.8535
+  Mt Lewis_1:
+    elevation (m): 987.0
+    latitude (deg): -16.55276
+    longitude (deg): 145.28038
+  Mt Lewis_2:
+    elevation (m): 994.0
+    latitude (deg): -16.55955
+    longitude (deg): 145.27363
+  Mt Lewis_3:
+    elevation (m): 995.0
+    latitude (deg): -16.55955
+    longitude (deg): 145.27364
+  Mt Lewis_4:
+    elevation (m): 996.0
+    latitude (deg): -16.55955
+    longitude (deg): 145.27363
+  Mt Lewis_5:
+    elevation (m): 1010.0
+    latitude (deg): -16.55277
+    longitude (deg): 145.28038
+  Mt Lewis_6:
+    elevation (m): 1015.0
+    latitude (deg): -16.55276
+    longitude (deg): 145.28038
+  Mt Lewis_7:
+    elevation (m): 1107.0
+    latitude (deg): -16.5166
+    longitude (deg): 145.27562
+  Mt Lewis_8:
+    elevation (m): 1210.0
+    latitude (deg): -16.5144
+    longitude (deg): 145.27206
+  Mt Lewis_9:
+    elevation (m): 1228.0
+    latitude (deg): -16.51219
+    longitude (deg): 145.26965
+  Mt Lewis_10:
+    elevation (m): 1232.0
+    latitude (deg): -16.5143
+    longitude (deg): 145.27204
+  Mt Lewis_11:
+    elevation (m): 1261.0
+    latitude (deg): -16.51028
+    longitude (deg): 145.27015
+  Mt Windsor_1:
+    elevation (m): 1253.0
+    latitude (deg): -16.217379
+    longitude (deg): 145.00162
+  Mt Windsor_2:
+    elevation (m): 1256.0
+    latitude (deg): -16.228646
+    longitude (deg): 145.01377
+  Mt Windsor_3:
+    elevation (m): 1260.0
+    latitude (deg): -16.217377
+    longitude (deg): 145.00164
+  Mt Windsor_4:
+    elevation (m): 1262.0
+    latitude (deg): -16.21733
+    longitude (deg): 145.00177
+  Mt Windsor_5:
+    elevation (m): 1286.0
+    latitude (deg): -16.217398
+    longitude (deg): 145.00179
+  Mt Windsor_6:
+    elevation (m): 1302.0
+    latitude (deg): -16.212347
+    longitude (deg): 144.999586
+contexts: .na
+traits:
+- var_in: leaf d13C
+  unit_in: '{delta}[ppth]'
+  trait_name: leaf_delta13C
+  entity_type: individual
```

Comment:
Checking that the branches sampled for each species were from different individuals (I assume that would be the case!)

Reply:
yes